machine_management/adding-rhel-compute.adoc

```diff
@@ -11,6 +11,13 @@ include::modules/rhel-compute-overview.adoc[leveloffset=+1]
 
 include::modules/rhel-compute-requirements.adoc[leveloffset=+1]
 
+include::modules/rhel-images-aws.adoc[leveloffset=+1]
+
+[NOTE]
+====
+When creating a {op-system-base} compute machine for AWS, ensure that the AMI is {op-system-base} 7. To do so, use the following command option: `--filters "Name=name,Values=RHEL-7.?*GA*"`.
```

Comment:
Does the filter value need to use {op-system-base} instead of hard-coding to RHEL? How does it work when the version for {op-system-base} may not be 7 when it is not RHEL?

Reply:
That's a variable in Asciidoc that populates the name to ensure consistency. In the browser, it renders as "RHEL."